src/services/api.test.js

```diff
@@ -49,4 +52,18 @@ describe('api', () => {
       expect(restaurants).toEqual(RESTAURANTS);
     });
   });
+
+  describe('fetchRestaurantInfo', () => {
+    beforeEach(() => {
+      mockFetch(RESTAURANTINFO);
+    });
+
+    it('returns restaurant information', async () => {
+      const restaurantInfo = await fetchRestaurantInfo({
+        restaurantId: 1,
+      });
+
+      expect(restaurantInfo).toEqual(RESTAURANTINFO);
```

Comment:
`restaurant`라는 말이 너무 많이 사용되는 것 같아요. `fetchRestaurantInfo`를 호출할 때는 Restaurant의 아이디로 조회한다는 것을 알기 때문에 `id`라고해도 충분할 것 같아요
```suggestion
      const info = await fetchRestaurantInfo({
        id: 1,
      });

      expect(info).toEqual(RESTAURANTINFO);
```

Reply:
네 해당 부분 수정하였습니다